eng/Microsoft.Extensions.targets

```diff
@@ -100,6 +100,11 @@
         Update="Microsoft.ProjectReunion.WinUI"
         Version="$(_MicrosoftProjectReunionVersion)"
     />
+    <PackageReference
+        Update="Microsoft.ProjectReunion.InteractiveExperiences"
+        Version="$(_MicrosoftProjectReunionVersion)"
+        NoWarn="NU1701"
+    />
```

Comment:
There seems to be a restore error because it can't figure out that net6-windows is like net5-windows, but just a bit more.

Reply:
@andrewleader Is this a known bug?

Reply:
I'll double check with the team, thanks Morten!